stac_generator/generator_factory.py

```diff
@@ -1,15 +1,16 @@
 from stac_generator.drone_stac_generator import DroneStacGenerator
-from stac_generator.sensor_stac_generator import SensorStacGenerator
 from stac_generator.generator import StacGenerator
+from stac_generator.sensor_stac_generator import SensorStacGenerator
+
+__all__ = ("StacGeneratorFactory",)
```

Comment:
Why did you add this?

Reply:
It's from one of the precommit hooks. It ensure that all files have __all__ i.e. choose what name to export

Reply:
Never seen a hook like that before. By default you export everything without a leading underscore so I don't see why its necessary.

Reply:
It's for type checking tools like mypy. It's more about explicitly forbidding private variables. Without __all__, when you import a private method, mypy would not say anything. If you don't put the private method in __all__ and try to import it, mypy will raise an issue. It's part of precommit hook, and it's just calling `make lint`. I can disable this if you want.